{
	"id": "159180",
	"key": "AC-3440",
	"fields": {
		"issuetype": {
			"id": "4",
			"description": "An improvement or enhancement to an existing feature or task.",
			"name": "Improvement",
			"subtask": false
		},
		"project": {
			"id": "12217",
			"key": "AC",
			"name": "Appcelerator - INBOX",
			"projectCategory": {
				"id": "10000",
				"description": "",
				"name": "Customer Service"
			}
		},
		"resolution": {
			"id": "1",
			"description": "A fix for this issue is checked into the tree and tested.",
			"name": "Fixed"
		},
		"resolutiondate": "2016-04-19T21:42:41.000+0000",
		"created": "2016-03-31T17:32:56.000+0000",
		"labels": [
			"feedback"
		],
		"versions": [
			{
				"id": "17038",
				"name": "Appcelerator Studio 4.5.0",
				"archived": false,
				"released": false
			}
		],
		"issuelinks": [],
		"assignee": {
			"name": "sdarda",
			"key": "sdarda",
			"displayName": "Sharif AbuDarda",
			"active": false,
			"timeZone": "Asia/Dhaka"
		},
		"updated": "2016-09-21T19:30:54.000+0000",
		"status": {
			"description": "The issue is considered finished, the resolution is correct. Issues which are closed can be reopened.",
			"name": "Closed",
			"id": "6",
			"statusCategory": {
				"id": 3,
				"key": "done",
				"colorName": "green",
				"name": "Done"
			}
		},
		"components": [],
		"description": "h3. in the tiapp.xml overview tab, it says that I am not part of the logged in organization even though, I am:\n\n1) Part of the organization (Appcelerator SE) \n2) Logged into studio with that organization as my active org\n3) Just created the app inside of studio\n",
		"attachment": [
			{
				"id": "58777",
				"filename": "Screen Shot 2016-03-31 at 12.31.14 PM.png",
				"author": {
					"name": "bgrantges@appcelerator.com",
					"key": "bgrantges",
					"displayName": "Bert Grantges",
					"active": false,
					"timeZone": "America/Chicago"
				},
				"created": "2016-03-31T17:32:59.000+0000",
				"size": 243358,
				"mimeType": "image/png"
			}
		],
		"flagged": false,
		"summary": "Tiapp.xml says i am not part of the organization",
		"creator": {
			"name": "bgrantges@appcelerator.com",
			"key": "bgrantges",
			"displayName": "Bert Grantges",
			"active": false,
			"timeZone": "America/Chicago"
		},
		"subtasks": [],
		"reporter": {
			"name": "bgrantges@appcelerator.com",
			"key": "bgrantges",
			"displayName": "Bert Grantges",
			"active": false,
			"timeZone": "America/Chicago"
		},
		"environment": null,
		"comment": {
			"comments": [
				{
					"id": "381227",
					"author": {
						"name": "sdarda",
						"key": "sdarda",
						"displayName": "Sharif AbuDarda",
						"active": false,
						"timeZone": "Asia/Dhaka"
					},
					"body": "Hello, Try logging out from studio. Close the studio and relaunch again and log in. See if the error happens. Thanks.",
					"updateAuthor": {
						"name": "sdarda",
						"key": "sdarda",
						"displayName": "Sharif AbuDarda",
						"active": false,
						"timeZone": "Asia/Dhaka"
					},
					"created": "2016-03-31T20:21:08.000+0000",
					"updated": "2016-03-31T20:21:08.000+0000"
				},
				{
					"id": "381870",
					"author": {
						"name": "bgrantges@appcelerator.com",
						"key": "bgrantges",
						"displayName": "Bert Grantges",
						"active": false,
						"timeZone": "America/Chicago"
					},
					"body": "This is still happening for me. I log out and log back in (from studio and CLI) and every time it tells me i am not apart of the organization.",
					"updateAuthor": {
						"name": "bgrantges@appcelerator.com",
						"key": "bgrantges",
						"displayName": "Bert Grantges",
						"active": false,
						"timeZone": "America/Chicago"
					},
					"created": "2016-04-06T16:22:29.000+0000",
					"updated": "2016-04-06T16:22:29.000+0000"
				},
				{
					"id": "382857",
					"author": {
						"name": "sdarda",
						"key": "sdarda",
						"displayName": "Sharif AbuDarda",
						"active": false,
						"timeZone": "Asia/Dhaka"
					},
					"body": "Hello, Try logging in through terminal. \r\n\r\nFirst, Run \r\nappc logout -D\r\n\r\nThen,\r\nappc login\r\n\r\nRegards,\r\nSharif.",
					"updateAuthor": {
						"name": "sdarda",
						"key": "sdarda",
						"displayName": "Sharif AbuDarda",
						"active": false,
						"timeZone": "Asia/Dhaka"
					},
					"created": "2016-04-15T20:08:46.000+0000",
					"updated": "2016-04-15T20:08:46.000+0000"
				},
				{
					"id": "396863",
					"author": {
						"name": "jfalcone",
						"key": "jfalcone",
						"displayName": "Joe Falcone",
						"active": true,
						"timeZone": "America/Los_Angeles"
					},
					"body": "We're seeing this problem with the latest Studio releases (4.7 and 4.7.1) with Pro subscriptions that have multiple seats.\r\n\r\nIt appears that Studio can get confused/corrupted/whatever and become unable to verify that you belong to an organization.\r\n\r\nThis has happened to myself (being unable to create Hyperloop projects) and my business partner (getting the \"you are not part of the organization\" message in tiapp).\r\nIs there something happening with the subscription validation service? \r\n\r\nI ask this because as a Paid user of Titanium, I am extremely upset about having these Paid services & features are unavailable to me and my business partner.",
					"updateAuthor": {
						"name": "jfalcone",
						"key": "jfalcone",
						"displayName": "Joe Falcone",
						"active": true,
						"timeZone": "America/Los_Angeles"
					},
					"created": "2016-09-21T19:30:54.000+0000",
					"updated": "2016-09-21T19:30:54.000+0000"
				}
			],
			"maxResults": 4,
			"total": 4,
			"startAt": 0
		}
	}
}